puma_motor_driver/src/multi_puma_node.cpp

```diff
@@ -48,104 +45,88 @@ MultiPumaNode::MultiPumaNode(const std::string node_name)
   this->get_parameter("gain.i", gain_i_);
   this->get_parameter("gain.d", gain_d_);
   this->get_parameter("gear_ratio", gear_ratio_);
-  joint_can_ids_ = this->get_parameter("joint_can_ids").as_integer_array();
+  joint_can_ids_    = this->get_parameter("joint_can_ids").as_integer_array();
   joint_directions_ = this->get_parameter("joint_directions").as_integer_array();
-  joint_names_ = this->get_parameter("joint_names").as_string_array();
-
-  RCLCPP_INFO(
-    this->get_logger(),
-    "Gear Ratio %f\nEncoder CPR %d\nFrequency %d\nGain PID %f,%f,%f\nCANBus Device %s",
-    gear_ratio_,
-    encoder_cpr_,
-    freq_,
-    gain_p_,
-    gain_i_,
-    gain_d_,
-    canbus_dev_.c_str()
-  );
+  joint_names_      = this->get_parameter("joint_names").as_string_array();
+
+  RCLCPP_INFO(this->get_logger(), "Gear Ratio %f\nEncoder CPR %d\nFrequency %d\nGain PID %f,%f,%f\nCANBus Device %s",
+              gear_ratio_, encoder_cpr_, freq_, gain_p_, gain_i_, gain_d_, canbus_dev_.c_str());
```

Comment:
If we like how the arguments were on different lines before we could add: `BinPackParameters: OnePerLine`

Reply:
I just tried this and it didn't seem to make any difference